spec/javascript/packages/document-capture/components/unknown-error-spec.jsx

```diff
@@ -132,4 +132,132 @@ describe('UnknownError', () => {
       expect(within(paragraph).getByText(/alternative message/)).to.be.ok();
     });
   });
+
+  context('there is a selfie quality/liveness failure', () => {
+    it('renders error message with errors', () => {
```

Comment:
I'm not sure I understand the difference between this test and the one below it. Do you mind clarifying?

Reply:
The difference is that in some situations we show "2 attempts remaining" text and some we don't. The expectations in the test pairs are subtly different to make that expectation happen.

Added some code comments about this [here](https://github.com/18F/identity-idp/pull/10042/commits/ff2b634a5c9a72f4ff05a281dac494d3e209f260).